man/journalctl.xml

```diff
@@ -362,6 +362,18 @@
         <xi:include href="version-info.xml" xpointer="v217"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-T</option></term>
+        <term><option>--exclude=<replaceable>SYSLOG_IDENTIFIER</replaceable></option></term>
```

Comment:
The long option should be --exclude-identifier or so as --exclude is too generic given this only applies to this single field

Reply:
Thank you! Done